components/brave_shields/browser/ad_block_regional_service_manager.cc

```diff
@@ -63,6 +63,9 @@ void AdBlockRegionalServiceManager::StartRegionalServices() {
     EnableFilterList(it->uuid, true);
   }
 
+  const bool cookie_list_touched =
+      local_state->GetBoolean(prefs::kAdBlockCookieListSettingTouched);
```

Comment:
Seems like a unit test to exercise this new preference could be useful.

Reply:
This is meant to be a pretty simple temporary change so we can use Griffin to evaluate whether or not it's worth pulling the cookie list into a better more permanent position, but point taken, I know how these "temporary" features go :sweat_smile:

I couldn't see an easy way to test this in a unit test, but I've added a browsertest.